src/components/Crews/ResultOverview.vue

```diff
@@ -28,7 +28,7 @@
                         </template>
                         <template v-slot:cell(FinishTime)="data">
                             <FinalTime
-                                :time="getFinishTime(data.item.times[0].times)"
+                                :time="getTotalTime(data.item.times[data.item.times.length - 1].times)"
```

Comment:
Waarom deze aanpassen?

Reply:
In de laatste colom is de calculated time nodig, dit is niet een finish time. Eigenlijk wil je weten of het een meerkamp is of niet en dan afh daarvan of een finish time zoeken of een calculated, maar enige manier om dat te checken is te kijken naar de length van data.item.times , en als die > dan 1 er vanuit gaan dat het meerkamp is.